django_switchuser/state.py

```diff
@@ -11,6 +11,8 @@ def __init__(self, request):
         self._reset()
 
     def _reset(self):
+        User = get_user_model()
```

Comment:
D'oh! I'm an idiot. It looked like the `User` was never being referenced because of the way the diff was shown. Merging and pushing this now! Thanks!